.github/workflows/integration-test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
       - name: Download KUTTL
         run: |
-          curl -fL https://github.com/kudobuilder/kuttl/releases/download/v${{ env.KUTTL_VERSION }}/kubectl-kuttl_${{ env.KUTTL_VERSION }}_linux_x86_64 -o kubectl-kuttl
+          curl -fL https://github.com/kudobuilder/kuttl/releases/download/${{ env.KUTTL_VERSION }}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64 -o kubectl-kuttl
```

Comment:
is the `#v` intended here?

Reply:
just checked - that seems to be correct indeed :)

Reply:
yes, the released file doesn't have the `v` in the name so i got rid of it